### PR TITLE
Speaker Coordination #8

### PR DESCRIPTION
**Pull request by tenkunkyab**
I have added a speaker-coordination guideline.
Thank you.

### CODE REVIEW

File: guidelines/speaker-coordination.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+----------------
+
+- Volunteers can go ahead and introduce the speaker to it's participants with the list of all requisites mentioned by the speaker.
+
+One week before conf
+----------------
+
+- The coordinator should make sure that the volunteers get to attend talks that they prefer and adjust the work force accordingly so that there are enough volunteers in an audi.
+- The speaker coordinator should contact the Volunteer coordinator and get the feedback form, if it is in paper.
+- Make Time card of 15 Minutes, 10 Minutes, 5 Minutes & Times Up.
```
Comment (kracekumar) [inline guidelines/speaker-coordination.rst]: It is `Time is up` not `Times up`.

File: guidelines/speaker-coordination.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+--------------
+
+- Make sure Projectors are set up.
+- Make sure there are at least three wireless mics for questionnaire and two portable mics for the speaker(s).
+
+Note for future coordinators
+------------
+
+- Contact all the speakers for the day and run them through the drill.
+- First day of Talk, do note that DgpLUG team will have a meeting around 11:30 am. So make sure that there are enough volunteers in audis.
```
Comment (kracekumar) [inline guidelines/speaker-coordination.rst]: Please remove the reference to dgplug. You can word it as "Volunteers may step out for other sessions/meetup discussion".

File: guidelines/speaker-coordination.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+- Allow participants to register for open space via mobile app.
```
Comment (kracekumar) [inline guidelines/speaker-coordination.rst]: This is a suggestion. Rather replace it as `Direct interested participants to open space coordinator`.

File: guidelines/speaker-coordination.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- Keep a mini-display port to HDMI/VGA handy. If possible one for each audi.
+- Do NOT forget to collect all the Feedback Forms.
+
+The Drill
+----------
+
+- Speaker has to be present at the audi at least 10 minutes before the scheduled start time.
+- Time cards, 15 minutes, 10 minutes, 5 minutes, and Times Up card are to be adhered.
```
Comment (kracekumar) [inline guidelines/speaker-coordination.rst]: Again it is `Time is up`.

**Comment by tenkunkyab** [comment]
Changed the sentences.

**Comment by tenkunkyab** [comment]
Let me know if any thing needs to be added/edited here.

**Comment by kracekumar** [comment]
@kirang89 thoughts ?

File: guidelines/speaker-coordination.rst [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+- Call for volunteers, at least 4 volunteers per audi should be sufficient. This excludes the Lead Speaker Coordinator as (s)he will have to run between different audi.
+- Gather contact details of all the volunteers.
+- Do ask the volunteers if there is any specific Workshop/Talk they would like to attend, so that there won't be any missing volunteers at the last minute.
+
+Three weeks before the conference
+----------------------
+
+- Divide the volunteers into different audis and make a list of all speakers in the respective audis.
+- Inform the volunteers to contact the respective Workshop Speakers requesting if there is any addition to the requisite already mentioned in the Proposal.
+- The additional requisite can contain steps to install packages xyz, etc.
```
Comment (kirang89) [inline guidelines/speaker-coordination.rst]: This should be merged with the previous point. I think the below points would be a sensible addition:

- Inform speaker to get a flash drive with the necessary materials (installers, dependencies) for the talk.
- Ask volunteers to preferably get a few flash drives as well, for backup (optional).

What say @kracekumar ?

Comment (kracekumar) [inline guidelines/speaker-coordination.rst]: Good one.

Comment (kirang89) [inline guidelines/speaker-coordination.rst]: Kindly commit with the following points @tenkunkyab and I'll be happy to merge your changes :)

**Comment by kirang89** [comment]
Looks good. Thanks for contributing! :)

**Comment by tenkunkyab** [comment]
I had fun and Thank you :)

**Pull request merged**